packages/bun-error/index.tsx

```diff
@@ -1439,7 +1390,12 @@ export function dismissError() {
       runtimeErrorController = null;
     }
 
-    while (pending.length > 0) pending.shift().stopThis = true;
+    while (pending.length > 0) {
+      const pendingFrame = pending.shift();
+      if (pendingFrame) {
```

Comment:
this branch is unnecessary because we already check `pending.length > 0`

Reply:
Fixed in https://github.com/oven-sh/bun/pull/753/commits/f4c3f73806e63f15ddb88d6893afbdfb2381a4b1